com.unity.cinemachine/Runtime/Components/Cinemachine3rdPersonFollow.cs

```diff
@@ -14,7 +14,7 @@ namespace Cinemachine
     /// </summary>
     [AddComponentMenu("")] // Don't display in add component menu
     [SaveDuringPlay]
-    [CameraPipelineAttribute(CinemachineCore.Stage.Body)]
+    [CameraPipeline(CinemachineCore.Stage.Body)]
```

Comment:
I agree with the change, but it will complicate the merge (I have renamed Body and Aim in my branch).  Consider deferring this change to later since it's just cosmetic and is not germane to the rest of this PR.